application/comit_node/src/http_api/routes/rfc003/handlers/get_action.rs

```diff
@@ -89,6 +91,8 @@ pub enum ActionResponseBody {
         amount: EtherQuantity,
         gas_limit: ethereum_support::U256,
         network: ethereum_support::Network,
+        #[serde(skip_serializing_if = "Option::is_none")]
+        valid_from: Option<Timestamp>,
```

Comment:
:x: I have some feedback on the field name here. I'd propose naming it `unavailable_until` because:

1. We usually use the inverse language when we talk about the refund action:

- "not yet able to do the refund"
- "have to wait for the refund action"

2. The field is optional. Does a missing `valid_from` mean it is not valid at all?

IMO, an optional negative constraint is more intuitive. If it is not there, you simply exclude it from the evaluation (that is what you did in the test).